packages/mux-player/src/index.ts

```diff
@@ -163,6 +164,20 @@ class MuxPlayerElement extends VideoApiElement {
     super();
 
     this.attachShadow({ mode: 'open' });
+
+    // If the custom element is defined before the <mux-player> HTML is parsed
+    // no attributes will be available in the constructor (construction process).
+    // Wait until initializing attributes in the attributeChangedCallback.
+    // If this element is connected to the DOM, the attributes will be available.
+    if (this.isConnected) {
```

Comment:
Ah, I thought it was going to be something like that. Forgot that `isConnected` is a thing.